packages/flutter/lib/src/widgets/banner.dart

```diff
@@ -330,6 +346,7 @@ class Banner extends StatelessWidget {
     properties.add(EnumProperty<TextDirection>('textDirection', textDirection, defaultValue: null));
     properties.add(EnumProperty<BannerLocation>('location', location));
     properties.add(EnumProperty<TextDirection>('layoutDirection', layoutDirection, defaultValue: null));
+    properties.add(DoubleProperty('height', height, showName: false));
```

Comment:
```suggestion
    properties.add(DoubleProperty('height', height));
```
showName should remain the default value in this case, refer to the documentation of DiagnosticsNode.showName:
```
  /// This could be set to false (hiding the name) if the value's description
  /// will make the name self-evident.
```
In this case, the number alone is not self-evident (unlike the color below it).